weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx

```diff
@@ -27,6 +34,18 @@ const CenterTableActionCellIcon = styled(LayoutElements.VStack)`
   }
 `;
 
+const DashboardTemplateItem = styled(LayoutElements.HStack)`
+  padding: 8px 12px;
+  border: 1px solid #dadee3;
```

Comment:
We should start trying to use named colors from the official design palette.

Reply:
fixed in next commit